packages/core-components/src/components/EmptyState/MissingAnnotationEmptyState.tsx

```diff
@@ -37,6 +37,7 @@ spec:
 
 type Props = {
   annotation: string;
+  link?: string;
```

Comment:
What do you think about calling this `readMoreUrl` instead? URL because that's a bit more precisely what it is, and with the `readMore*` prefix to give it a bit more context. Another option could also be a plain `url`.